### PR TITLE
OO|ME: Do not render table view unless it is active

### LINKED ISSUE #131: Do not render table view unless active

### PR DESCRIPTION
**Pull request by okal**
Closes #131

### CODE REVIEW

File: src/hatti/views/table.cljs [hunk 1]
```diff
@@ -416,44 +416,52 @@
...
-      (let [old-data (get-in (om/get-props owner) [:data])
-            new-data (get-in next-props [:data])
-            {:keys [grid dataview]} (om/get-state owner)]
-        (when (not= old-data new-data)
-          (if (empty? old-data)
+  (let [active? (-> app-state
+                    :views
+                    :active
+                    set
+                    (contains? :table))]
```
Comment (pld) [inline src/hatti/views/table.cljs]: feels like we should stick w/the convention and use `in?` here

Comment (okal) [inline src/hatti/views/table.cljs]: Thought so too, but was surprised not to find it in Hatti. It's in Chimera now, right?

Comment (okal) [inline src/hatti/views/table.cljs]: Found it. Pushing in a bit.

Comment (pld) [inline src/hatti/views/table.cljs]: Yup

Comment (pld) [inline src/hatti/views/table.cljs]: Hmm, we need public docs for chimera, so you don't have to go lurking through the source code to find these things

**Pull request merged**